libs/model/src/main/java/de/cyface/model/Measurement.java

```diff
@@ -89,6 +95,64 @@ public Measurement(final MetaData metaData, final List<Track> tracks) {
         this.tracks = new ArrayList<>(tracks);
     }
 
+    public Measurement(final List<TrackBucket> buckets) {
```

Comment:
These two methods where in `collector.api.MeasurementIterator` before but now I also need to create a `Measurement` from `backend.libs.mongo-reader` for the Flink Surface Pipeline to load already deserialized data ("TrackBuckets").

In order to now have to add `libs/api` dependency to `libs/mongo-reader` I moved this to the `Measurement` model, which makes more sense I think.